src/Clipper.jl

```diff
@@ -534,6 +542,10 @@ function Base.length(c::__ClipperPolyTree)
     @cxx c->Total()
 end
 
+function child_count(c::__ClipperPolyTree)
+    @cxx c->ChildCount()
+end
+
```

Comment:
I wrote a test for `child_count`.

I am struggling with implementing `getindex`. Here is what I tried:

```
@inline function Base.getindex(p::__ClipperPolyTree, i::Integer)
    icxx"$p[$i-1];"
end
```

I get the following error when I try to call `getindex`

```
ERROR: LoadError: MethodError: `getindex` has no method matching getindex(::Cxx.CppValue{Cxx.CppBaseType{symbol("ClipperLib::PolyTree")},(false,false,false)}, ::Int64)
Closest candidates are:
  getindex(::(Any...,), ::Int64)
  getindex(::(Any...,), ::Real)
  getindex{T}(::FloatRange{T}, ::Integer)
```